stix_shifter_modules/reaqta/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -1379,7 +1379,7 @@
       },
       "etwWorkstation": {
         "key": "x-oca-asset.host_id",
-        "object": "etw_asset"
+        "object": "asset"
```

Comment:
Does this need to match etwWorkstationName below?  That also uses `etw_asset`.

Reply:
ohh that's a mistake. I should have kept the `etw_asset` object name. reverting back